File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2982.rb
n = gets.to_i

soma = 0
for i in 1..n do
    linha = gets
    ls = linha.split(' ')
    
    e = ls[0]
    v = (ls[1]).to_i
    
    if(e == "G") then
        soma=soma-v
    else
        soma=soma+v
    end
end
        
if(soma >= 0) then
    puts("A greve vai parar.")
else
    puts("NAO VAI TER CORTE, VAI TER LUTA!")
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2982.cs
using System;

class URI
{
    static void Main() {
        int soma = 0, n = int.Parse(Console.ReadLine().Trim());
        
        for(int i = 0; i < n; i++){
            string linha = Console.ReadLine();
            if (string.IsNullOrEmpty(linha))
                break;
            string[] ls = linha.Split(' ');
            
            string e = ls[0];
            int v = int.Parse(ls[1]);

            if(e == "G"){
                soma-=v;
            } else{
                soma+=v;
            }
        }
        
        if(soma>=0){
            Console.Write("A greve vai parar.\n");
        } else{
            Console.Write("NAO VAI TER CORTE, VAI TER LUTA!\n");
        }
        
    }
}
